libnymea-core/hardware/modbus/modbusrtumasterimpl.cpp

```diff
@@ -260,8 +260,8 @@ ModbusRtuReply *ModbusRtuMasterImpl::readCoil(int slaveAddress, int registerAddr
         emit reply->finished();
     });
 
-    connect(modbusReply, &QModbusReply::errorOccurred, modbusReply, [=](QModbusDevice::Error error){
-        qCWarning(dcModbusRtu()) << "Read coil request finished with error" << error << modbusReply->errorString();
+    connect(modbusReply, &QModbusReply::errorOccurred, reply, [=](QModbusDevice::Error error){
+        qCWarning(dcModbusRtu()) << "Read coil register request finished with error" << error << modbusReply->errorString();
```

Comment:
Coil is not a `register`, it's can only give you the `status`, like a bit. 
See function code table as refference.